docusaurus/docs/reactnative/guides/message_input_customization.mdx

```diff
@@ -229,4 +232,164 @@ const additionalTextInputProps = useMemo(() => {
 >
   ...
 </Channel>
-```
\ No newline at end of file
+```
+
+## Replacing AttachmentPicker With Native Image Picker
+
+Attachment picker opens up as bottom sheet by default.
```

Comment:
`Attachment picker is rendered/displayed/opened inside of a bottom sheet by default.`?